app/views/users/edit.html.erb

```diff
@@ -23,7 +23,7 @@
   <div class="crayons-layout crayons-layout--header-inside crayons-layout--2-cols crayons-layout--limited-l">
     <div class="crayons-page-header">
       <% if @tab == "organization" && @organizations.present? %>
-        <h1 class="crayons-title">Settings for
```

Comment:
Should this (non-localized) text have been replaced with a translation key?

Reply:
I mean, the equivalent string is added a few lines below:
https://github.com/forem/forem/blob/6ee334f10e25f47c9dc9772291c8b35572bba2b1/app/views/users/edit.html.erb#L31
Currently as I see the line produces a duplicated message. 
![image](https://user-images.githubusercontent.com/84892012/145911486-4c7ed212-f107-46b4-b878-b46c26bd2018.png)